main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genre_list = ["Western", "Action", "Dramma"]
+    actors_list = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]
```

Comment:
Using a list of tuples is better.

Tuples are often used to represent fixed collections of items. In your case, each actor's name is a fixed pair of first and last names. Using tuples can make it clearer that these pairs are fixed sets of data.